CHANGELOG.md

```diff
@@ -1,4 +1,18 @@
 # Changelog
+## [1.2.1] - 2021-03-24
```

Comment:
This should probably be 1.3.0 since there are new features and bigger changes for inclusive language